projects/packages/stats/src/class-tracking-pixel.php

```diff
@@ -61,7 +73,16 @@ public static function build_view_data() {
 		} else {
 			$post = '0';
 		}
-		return compact( 'v', 'blog', 'post', 'tz', 'srv' );
+		$view_data = compact( 'v', 'blog', 'post', 'tz', 'srv' );
+		foreach ( self::TRACKED_UTM_PARAMETERS as $utm_parameter ) {
+			// phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
When we ignore specific PHPCS rules, I often recommend adding a comment explaining why. It makes things a bit easier later on, when coming back to a file that was edited a long time ago ; it feels safer knowing that we added the exception knowingly.

PHPCS allows adding comments like so:

```suggestion
			// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- we clear the parameter as we output it below.
```

Reply:
Thank you for the suggestion! I've added a comment explaining why. UTM parameters are coming from external websites by design, so we don't control them and can't add a nonce here.